.github/workflows/tmt-tests.yml

```diff
@@ -0,0 +1,51 @@
+---
+name: TMT Tests
+
+on:
+  push:
+    branches: 
+      - main
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      plan_filter:
+        description: 'Plan filter name, ie: tag:smoke'
+        required: false
+        default: ''  # defaults to an empty value if nothing is provided
+
+jobs:
+  tmt-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - uses: actions/setup-go@v5
+        with:
+          go-version: 'stable'
+
+      - name: Install dependencies
+        run: |
+          set -x -e -o pipefail
+          sudo apt-get update
+          sudo apt-get install -y podman libblkid-dev
+          pip install --user tmt
+
+      - name: Build Ignition
+        run: |
+          set -x -e -o pipefail
+          make all
+
+      - name: Run TMT tests
+        run: |
+          set -x -e -o pipefail
+          PLAN_FILTER="${{ github.event.inputs.plan_filter }}"
```

Comment:
Thats awesome! great work. So from my understanding as we add new tests they will automatically be incorporated to the CI?